src/browser-installer/chromium/revisions/win32.ts

```diff
@@ -8,7 +8,6 @@ export default {
     79: 707225,
     80: 722370,
     81: 737194,
-    82: 750023,
```

Comment:
For `chrome for testing` versions (upstream) we dont need to manually download drivers, as this can be handled by `@puppeteer/browsers`

But for older versions we need to download chromedriver ourselves. We only can download chromedriver manually if we know its full version: `${number}.${number}.${number}.${number}`
To get latest chrome driver version for chrome/(ium) milestone we use this API:
`https://chromedriver.storage.googleapis.com/LATEST_RELEASE_<N>`, where `N` - chrome/(ium) milestone
Example: https://chromedriver.storage.googleapis.com/LATEST_RELEASE_112 (resolves 112.0.5615.49)

For unknown reason, this API just does not work when `N = 82`

Reply:
Renamed to ".json"